docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -0,0 +1,176 @@
+
+PKI Side
+^^^^^^^^
+
+You will first need to log into the PKI:
+
+* Address: https://192.168.1.5:9191
+* Login: admin
+* Password: packetfence
+
+After you logged in there are few steps to follow to have the PKI working for an EAP-TLS configuration with PacketFence. You will need to create the following items: 
+
+* Certificate of Authority.
+* Profiles(for each purpose, ie Server/Client auth are 2 different profiles).
+* Manual certificate generation.(for the RADIUS server)
+
+To help you out through the configuration, the first login will ask you to create a Certificate of Authority.
+
+Certificate of Authority
+^^^^^^^^^^^^^^^^^^^^^^^^
+ 
+* Key type, size and digest: we advise to use the following RSA, 2048, sha1.
+* Key Usage and Extended Key Usage can be filled if necessary, more information about Key Usage and Extended Key Usage are available from the RFC5280, parts 4.2.1.3 and 4.2.1.12.
+* Days: Number of validity days, ie 3y = 1068.
+
+CAUTION: Remember that after the expiration date of your Certificate of Authority is passed, every certificate generated by it will not be valid anymore.
+
+Profile
+^^^^^^^
+
+Creation of a profile:
+
+* Name: MyClientProfile
+* Ca: The certificate of authority you created earlier
+* Crl path: Certification revokation list path, a file will be created when you fill this.
+* Validity: Number of validity days, ie 3y = 1068
+* Key type, size and digest: we advise to use the following RSA, 2048, sha1
+* Key Usage: Empty 
+* Extended Key Usage: 
+** Server certificate authentication profile: TLS Web Server Authentication, 
+** Client certificate authentication profile: TLS Web Client Authentication, 
+
+Information about Key Usage and Extended Key Usage are available from RFC5280, parts 4.2.1.3 and 4.2.1.12.
+
+* The P12 mail setup is mandatory at least for the server authentication profile. This will format the mail send with your certificate if you use *send certificate* button, to send it manually. 
+** If your mail alert are working with PacketFence you should use the following:
+*** P12 smtp server: 127.0.0.1
+*** Tick P12 mail password
+
+For the header, from, subject and footer this part is the customization of the mail, it is configurable as you want it for your company. An example of value would be:
+
+* Form: pki@packetfence.pki
+* Subject: Certificate for EAP-TLS connection
+* Header: Below you will find your certificate and it's password to install it.
+* Footer: This email has been generate automatically, please do not reply.
+
+RADIUS server authentication certificate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Generate a certificate from the profile you defined as the server authentication profile. This certificate will need to be installed and configured as the certificate for authenticate RADIUS. Also it will be necessary to install and trust the CA on PacketFence.
+
+CAUTION: We advise to use a long validity for your RADIUS server certificate to avoid renew it often.
```

Comment:
You should show the user how to generate the certificate and give him a good timeframe (like 10y)